web/spacewalk-web.spec

```diff
@@ -199,8 +184,6 @@ mkdir -p %{buildroot}%{_sysconfdir}/httpd/conf
 mkdir -p %{buildroot}%{_sysconfdir}/cron.daily
 
 install -m 644 conf/rhn_web.conf %{buildroot}%{_datadir}/rhn/config-defaults
-install -m 644 conf/rhn_dobby.conf %{buildroot}%{_datadir}/rhn/config-defaults
```

Comment:
@rjmateus `conf/rhn_dobby.conf` can also be removed with this PR

Reply:
go catch @mcalmer, thank you.
I have just removed the file